trading_report/src/verify.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+import { writeFile } from 'node:fs';
+import { PublicKey } from '@solana/web3.js';
+
+async function verify() {
+  console.log('Verify addresses are valid', new Date().toUTCString());
+
+  const data = parseCSVFromFile("./address.csv");
+  let verifiedAddress = [];
+  for(const address of data){
+    try{
+      const pubKey = new PublicKey(address);
+      if (PublicKey.isOnCurve(pubKey)) {
```

Comment:
Can we check that it is an active wallet with non-zero SOL in it?

Reply:
We shouldnt care whether it has SOL in it right?

Reply:
Non-zero SOL would just indicate that it is a real user who is able to pay gas, not just some generated keypair. We dont need to check that, but would avoid waste.

Reply:
Would potentially piss off any new Solana user who signed up & I did explicitly say in the form you don't need to have funded the account so holding off. Would be a good potential catch for sybil attackers but already filtered those